### DISCUSSION TITLE
LAN training vs parameter tradeoff issue

### DISCUSSION THREAD

**Discussion by jacobsellers** [lnccbrown/HSSM#947] [Q&A]
Hi all,

I've been using the ssm-simulators + LANfactory packages to train a LAN for use as a custom likelihood in HSSM. Everything up through training the LAN has gone smoothly to my knowledge. I tried simulating some data and fitting the model using HSSM, but the fitted parameter values were pretty far off from the true values. Everything about the fit seems to point to parameter tradeoffs -- some correlations between posterior samples in pair plots while the PPC lines up with the data and the model converged. Additionally, the learned likelihood function aligns with simulated data (for this set of parameter values at least)

However, I was curious when I can be sure that I should treat this as a parameter tradeoff issue with the model rather than an issue with the LAN? Given the model can still account for simulated data (just with different parameters), I'm leaning toward it being a sampling issue. So I'm just wondering if there is any way to be more sure that the LAN has learned the likelihood function before concluding there is a sampling issue. I figured this would also be helpful for future uses of LANs.

For reference, I trained the LAN on 120k parameter combinations with 2000 samples per parameter set. I trained 4 different model architectures for 60 epochs (85/15 train/test split) and selected the architecture with the lowest validation loss. Validation loss for each architecture was pretty comparable and seemed to have leveled off at the end of training too.

Thanks for any help!

Jacob
